data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/AbstractLogstashPluginAttributesMapperTest.java

```diff
@@ -62,4 +66,18 @@ void mapAttributes_with_custom_attributes_invokes_mapCustomAttributes_Test() {
         verify(abstractLogstashPluginAttributesMapper).mapCustomAttributes(logstashAttributes, mappings, pluginModels.get(0).getPluginSettings());
     }
 
+    @Test
+    void mapAttributes_with_default_settings_Test() {
+        AbstractLogstashPluginAttributesMapper abstractLogstashPluginAttributesMapper = Mockito
+                .spy(AbstractLogstashPluginAttributesMapper.class);
+        final String defaultSettingAttributeName = UUID.randomUUID().toString();
+        final String defaultSettingAttributeValue = UUID.randomUUID().toString();
+        when(mappings.getDefaultSettings()).thenReturn(Collections.singletonMap(defaultSettingAttributeName, defaultSettingAttributeValue));
+
+        List<PluginModel> pluginModels = abstractLogstashPluginAttributesMapper.mapAttributes(Collections.emptyList(), mappings);
+
+        assertThat(pluginModels, Matchers.notNullValue());
+        assertThat(pluginModels.size(), Matchers.equalTo(1));
+        assertThat(pluginModels.get(0), Matchers.notNullValue());
```

Comment:
This test does not verify that the `PluginModel` has the correct settings.

The test should get the first plugin model and then verify that it has a single setting value. The setting should have the `defaultSettingAttributeName` with the correct value.